style/styles_stories.css

```diff
@@ -0,0 +1,260 @@
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stories-item__link {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 12px;
+  text-decoration: none;
+  color: #000;
+}
+
+.stories-item__link::after {
+  content: url("../pic/arrow_stories.png");
+  padding-left: 7px;
+}
+
+.stories-item__date {
+  font-family: "Gotham Pro Bold", sans-serif;
+  font-size: 12px;
+  opacity: 0.3;
+}
+
+@media (max-width: 1024px) {
+  .stories {
+    padding-top: 78px;
+  }
+
+  .pic-first {
+    display: inline;
+  }
+
+  .pic-first-big {
+    display: none;
+  }
+
+  .stories-item__article {
+    font-size: 13px;
+    line-height: 16px;
+    padding-bottom: 24px;
+  }
+
+  .stories-item__link::after {
+    content: url("../pic/arrow_stories1024.png");
+    padding-left: 0;
+  }
+}
+
+@media (max-width: 960px) {
+  .stories-item__pic {
+    max-height: 127px;
+  }
+}
+
+@media (max-width: 820px) {
+  .stories-item__pic {
+    max-height: 107px;
+  }
+}
+
+@media (max-width: 768px) {
+  .stories {
+    margin-top: -1px;
+    margin-bottom: -1px;
+    background-color: #000;
+    color: #fff;
+    padding-top: 82px;
+  }
+
+  .stories__title {
+    font-size: 30px;
+    margin-bottom: 42px;
+  }
+
+  .item-third {
+    display: none;
+  }
+
+  .stories__block {
+    max-width: 681px;
+    justify-content: space-between;
+    margin: 0 auto;
+  }
+
+  .item-first {
+    width: 49%;
+  }
+
+  .item-second {
+    width: 46%;
+  }
+
+  .pic-first {
+    width: 94%;
+  }
+
+  .stories-item__pic {
+    padding-bottom: 28px;
+    max-height: 165.54px;
+    height: 100%;
+  }
+
+  .stories-item__title {
+    font-size: 25px;
+    padding-bottom: 8px;
+  }
+
+  .stories-item__article {
+    font-size: 20px;
+    padding-bottom: 36px;
+    line-height: 25px;
+  }
+
+  .item-first .stories-item__article {
+    width: 94%;
+  }
+
+  .stories-item__link {
+    color: #fff;
+    font-size: 20px;
+  }
+
+  .stories-item__link::after {
+    content: url("../pic/arrow_stories768.png");
+    padding-left: 12px;
+  }
+
+  .stories-item__date {
+    font-size: 15px;
+  }
+}
+
+@media (max-width: 700px) {
+  .stories__block {
+    max-width: 610px;
+  }
+
+  .stories-item__title {
+    font-size: 23px;
+  }
+
+  .stories-item__article {
+    font-size: 15px;
+  }
+
+  .stories-item__link {
+    font-size: 16px;
+  }
+}
+
+@media (max-width: 655px) {
+  .item-second {
+    display: none;
+  }
+
+  .stories-item {
+    width: 70%;
+  }
+
+  .stories__block {
+    justify-content: center;
+  }
+
+  .stories-item__pic {
+    max-height: 200px;
+    height: auto;
+  }
+}
+
+@media (max-width: 420px) {
+  .stories__title {
+    font-size: 20px;
+    margin-bottom: 26px;
+  }
+
+  .stories-item {
+    width: 74.5%;
+  }
+
+  .stories-item__pic {
+    padding-bottom: 27px;
+    width: 100%;
+  }
+
+  .stories-item__title {
+    font-size: 20px;
+    padding-bottom: 10px;
+  }
+
+  .item-first .stories-item__article {
+    width: 100%;
+  }
+  .stories-item__article {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".stories-item__article" to come before selector ".item-first .stories-item__article" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/AngelinaAxion/pullRequest?prid=5638582)